src/pyuvdata/uvbase.py

```diff
@@ -870,3 +870,78 @@ def use_future_array_shapes(self):
         This function has been deprecated, and will result in an error in version 3.2.
         """
         self._set_future_array_shapes(True)
+
+    def _select_along_param_axis(self, param_name, ind_arr):
+        """
+        Downselect values along a parameterized axis.
+
+        This method should not be called directly by users; instead, it is called by
+        various selection-related functions for selecting a subset of values along
+        a given axis, whose expected shape is given (at least in part) by a named
+        parameter within the object (e.g., "Nblts", "Ntimes", "Nants"). Additionally,
+        this method will recalculate the value for the named parameter given the input
+        indexing array.
+
+        Parameters
+        ----------
+        param_name : str
+            Name of the parameter, which is used to define the expected shapre of other
```

Comment:
shapre -> shape

Reply:
Fixed in bee13af03969accbf712e9db80f799bd0d142a9b